Docker build failing with this message:

> 2316.1 /opt/roofer/src/extra/io/VectorReaderOGR.cpp:215:65: error: invalid conversion from 'const OGRSpatialReference*' to 'OGRSpatialReference*' [-fpermissive]
> 2316.1   215 |       if (OGRSpatialReference* layerSRS = poLayer->GetSpatialRef()) {
> 2316.1       |                                           ~~~~~~~~~~~~~~~~~~~~~~^~
> 2316.1       |                                                                 |
> 2316.1       |                                                                 const OGRSpatialReference*

Seen this already in City4CFD: https://github.com/tudelft3d/City4CFD/pull/52/commits/3665268707790250b845979a2ebee9507caddf73

--- a/rooferpy/README.md
+++ b/rooferpy/README.md
@@ -8,4 +8,4 @@ cmake --preset vcpkg-with-bindings -S . -B build_python
 cmake --build build_python
 ```
 
-The rooferpy library will be located in `build_python/rooferpy/rooferpy.cpyton-<version-and-system>.so`. Import the .so file (e.g. place it in the same folder as .py script) to use roofer python API.
+The rooferpy library will be located in `build_python/rooferpy/roofer.cpyton-<version-and-system>.so`. Import the .so file (e.g. place it in the same folder as .py script) to use roofer python API.

--- a/src/extra/io/VectorReaderOGR.cpp
+++ b/src/extra/io/VectorReaderOGR.cpp
@@ -212,7 +212,7 @@ namespace roofer::io {
       if (poLayer == nullptr) {
         throw(rooferException("[VectorReaderOGR] Layer is not open"));
       }
-      if (OGRSpatialReference* layerSRS = poLayer->GetSpatialRef()) {
+      if (auto* layerSRS = poLayer->GetSpatialRef()) {
         if (!srs->is_valid()) {
           char* pszWKT = NULL;
           layerSRS->exportToWkt(&pszWKT);

